src/views/Home.vue

```diff
@@ -230,6 +236,10 @@ export default {
         .join('-');
 
       window.location.hash = `plan/${encodedPlan}`;
+
+      if (encodedPlan !== []) {
```

Comment:
So the user, when checking out another users plan by opening the url, completely uses his locally saved plan?

Reply:
no, the locally saved one will be replaced. Maybe we need an explicit save button :)

Reply:
still better to have the behavior introduced in this PR than nothing, so I’d still like to merge it

Reply:
I'd personally rather have a no feature than a fundamentally broken feature, but that's your call.

Reply:
I don't think it's broken. It doesn't break any current user flows and may help some people. Calling it fundamentally broken is a bit much.